driver-core/src/main/com/mongodb/operation/CommandOperationHelper.java

```diff
@@ -499,10 +499,7 @@ public R call(final ConnectionSource source, final Connection connection) {
                                     readPreference, commandResultDecoder, binding.getSessionContext()),
                                     connection.getDescription().getServerAddress());
                         } catch (MongoException e) {
-                            if (binding.getSessionContext().hasActiveTransaction()) {
-                                throw e;
-                            }
-                            throw originalException;
+                            throw e;
```

Comment:
Since we're now just throwing `e` here, the whole catch block should just be removed, as the effect is the same

Reply:
I have removed the block.